### ISSUE TITLE
Inconsistent spelling of structure comment for sample storage container

### ISSUE THREAD

**Issue by ramonawalls** [GenomicsStandardsConsortium/mixs#409]
All of the older packages that include `sample storage container` (mixs:000755) use `sample_store_cont` as the structured comment, whereas the new farm packages (food-human foods, food-animal and animal feed, food-food production facility) use `sample_stor_cont`. 

We should make them all `sample_store_cont`. 

Need to reach out to the farm group before making this change.

**Comment by only1chunts** [comment]
I agree with @ramonawalls 
## Proposal
All instances of MIXSID:0000755 should use the structured comment name "sample_store_cont"

**Comment by mslarae13** [comment]
Hmm...
sample storage container - `sample_store_cont` - mixs:000755 doesn't exist

What does exist:
sample storage location - `samp_store_loc` - mixs:0000755
- https://genomicsstandardsconsortium.github.io/mixs/0000755/

sample storage device - `samp_stor_device` - mixs:0001228/
- https://genomicsstandardsconsortium.github.io/mixs/0001228/


@turbomam can you chime in here on what happened?

**Comment by lschriml** [comment]
Looking at MIxS 6.0 release and MIXS ID 
https://docs.google.com/spreadsheets/d/1p5Ziciznkk9er99aPhc7ed8NDE0oUKw0rrBlEOYUtiA/edit?gid=0#gid=0
     --  I confirm that mixs:0000755.  is: 
              sample storage location - samp_store_loc

in food packages: 
                   the short name is; samp_stor_loc
   --> needs to be updated to samp_store_loc

**Comment by mslarae13** [comment]
I looked in mixs.yaml and in the documentation
`samp_stor_loc` is something @turbomam must have fixed when LinkML was implemented. Because all the food extensions have `samp_store_loc` now.

@turbomam  could you take a second look and close this issue if you find the same result?

**Comment by turbomam** [comment]
I agree. There are 20 appearances of "`- samp_store_loc`" (indicating that `samp_store_loc` has been associated with 20 classes) and zero appearances of `samp_stor_loc`
